lib/kdump_utils.pm

```diff
@@ -116,6 +132,21 @@ sub prepare_for_kdump {
         zypper_call("rr $snapshot_debuginfo_repo");
         return;
     }
+    if (is_alp) {
+        my $repo_url = 'https://updates.suse.com/SUSE/Products/ALP-Dolomite/1.0/x86_64/product_debug/';
```

Comment:
Hardcoded repositories is not optimal. I would use variable define at test suite level `REPO_TRANSACTIONAL_DEBUG' with ibs url.

Reply:
This is the URL I was pointed at when asking for debug repos for ALP. I have no idea where I would find it on IBS. I found https://download.suse.de/ibs/SUSE:/ALP:/Products:/Dolomite:/1.0/standard/ but it doesn't seem to have the debug repo.

Reply:
To answer my own question: Maybe this repository http://download.suse.de/ibs/SUSE:/ALP:/Source:/Standard:/Core:/1.0:/Build/standard/x86_64/ could be used for ALP. But I made it use `REPO_TRANSACTIONAL_DEBUG` as suggested anyways so this question will only be relevant once we add the test to the yaml schedule.